src/maximize.ts

```diff
@@ -468,13 +469,14 @@ export function maximizeCached(
     applyCached(cacheEntry, fullOptions);
     if (verifyCached(cacheEntry)) {
       logger.info(`Equipped cached ${objective}`);
-      return;
+      return true;
```

Comment:
Do we assume that the outfit succeeds?

Reply:
Yeah but we should fix that separately (and anyway we verify after)